### PR TITLE
Sb8241

### PR DESCRIPTION
**Pull request by amolnin**

### CODE REVIEW

File: common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ private UserRequestValidator() {}
    *
    * @param userRequest Request
    */
-  public static void validateCreateUser(Request userRequest) {
+  public static void validateCreateUserRequest(Request userRequest) {
```
Comment (bvinayakumar) [inline common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java]: This PR has several files which are not related (e.g. unit test files). Can you check and revert once done?

**Comment by amolnin** [comment]
deleting becoz branch base was incorrect
